dojo/tools/dependency_check/parser.py

```diff
@@ -238,7 +246,21 @@ def get_finding_from_vulnerability(self, dependency, related_dependency, vulnera
                                      'source: {1}\n' \
                                      'url: {2}\n\n'.format(name, source, url)
 
-        mitigation = 'Update ' + component_name + ':' + component_version + ' to at least the version recommended in the description'
+        if related_dependency is not None:
+            tags.append("related")
+
+        if vulnerability.tag == "{}suppressedVulnerability".format(namespace):
+            if notes == "":
+                notes = "Document on why we are suppressing this vulnerability is missing!"
+                tags.append("missingdoc")
```

Comment:
not sure if we want to tag this also, but if we do, can we make it something more relevant, i.e. no_suppression_reason or something?

Reply:
and can we add a finding to the unit test to cover this case?

Reply:
> not sure if we want to tag this also, but if we do, can we make it something more relevant, i.e. no_suppression_reason or something?

I think so too. `missingdoc` is too generic. I'll change it with `no_suppression_document`.

Reply:
> and can we add a finding to the unit test to cover this case?

There is a test for that. However it's not for the tag, it's for the Mitigation field. Well, the tags are... complicated :) I will have to spend some time for this.